### PR TITLE
Marketplace bug2

### PR DESCRIPTION
**Pull request by namansleeps**
### Description
Resolved unusual loading appearance in agent template. Made minor fixes to loading

### Related Issues
No Issues

### Solution and Design
Loading has been added in agent template and minor bug fixed in toolkit marketplace page

### Test Plan
Frontend changes made and showed to product

### Type of change
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Docs update

### Checklist
- [x] My pull request is atomic and focuses on a single change.
- [x] I have read the contributing guide and my code conforms to the guidelines.
- [ ] I have documented my changes clearly and comprehensively.
- [x] I have added the required tests.

### CODE REVIEW

**Review by DMTarmey — COMMENTED** [review]
Update marketplace

File: gui/pages/Content/Marketplace/MarketTools.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ export default function MarketTools(){
   const [loadingText, setLoadingText] = useState("Loading Toolkits");
 
   useEffect(() => {
-    loadingTextEffect('Loading Templates', setLoadingText, 500);
+    loadingTextEffect('Loading Toolkits', setLoadingText, 500);
```
Comment (nborthy) [inline gui/pages/Content/Marketplace/MarketTools.js:16]: keep it Loading Templates

File: gui/pages/Content/Marketplace/MarketTools.js [hunk 2]
```diff
@@ -13,7 +13,7 @@ export default function MarketTools(){
   const [loadingText, setLoadingText] = useState("Loading Toolkits");
```
Comment (nborthy) [inline gui/pages/Content/Marketplace/MarketTools.js:13]: Change it to Loading Templates

**Review by nborthy — APPROVED** [review]

File: gui/pages/Content/Marketplace/MarketTools.js (hunk 2 above)
Comment (namansleeps) [reply gui/pages/Content/Marketplace/MarketTools.js:13]: Adarsh said to change it to loading toolkits

**Pull request merged**